cwe_checker_rs/src/analysis/backward_interprocedural_fixpoint/mod.rs

```diff
@@ -43,8 +56,8 @@ pub trait Context<'a> {
     /// Transition function for in-program calls.
     fn update_callsite(
         &self,
-        value_after_call: Option<&Self::Value>,
-        fallthrough_value: Option<&Self::Value>,
+        target_value: Option<&Self::Value>,
+        return_value: Option<&Self::Value>,
```

Comment:
Hmm, I am not sure whether these new names are better. But since I don't know better names either, we probably should simply add a clarifying comment to the doc comment of `update_callsite`.